examples/twittermap/web/public/javascripts/map/pointmap.js

```diff
@@ -88,13 +88,13 @@ angular.module('cloudberry.map')
       }
 
       $scope.loadGeoJsonFiles(onEachFeature);
-      
+
```

Comment:
Why is this empty line required?

Reply:
It is because Qiushi provides me with new sample data, so we have change the application.conf file to make it work. I think the empty line should be the format issue. Let me see if I can adjust it.

Reply:
Please remove the application.conf change in your change. You can do this by overwriting the current one to your commit.